openresty/Dockerfile-alpine

```diff
@@ -0,0 +1,253 @@
+    REAL_IP_HEADER="X-REAL-IP" \
+    REAL_IP_RECURSIVE="on" \
+    PROXY_TIMEOUT=60s \
+    PROXY_SSL_CERT=/usr/local/openresty/nginx/conf/server.crt \
+    PROXY_SSL_CERT_KEY=/usr/local/openresty/nginx/conf/server.key \
+    PROXY_SSL_DH_BITS=2048 \
+    PROXY_SSL_PROTOCOLS="TLSv1.2 TLSv1.3" \
+    PROXY_SSL_CIPHERS="ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384" \
+    PROXY_SSL_PREFER_CIPHERS=off \
+    PROXY_SSL_VERIFY=off \
+    PROXY_SSL_OCSP_STAPLING=off \
+    SERVER_NAME=localhost \
+    SSL_PORT=443 \
+    TIMEOUT=60s \
+    WORKER_CONNECTIONS=1024 \
+    # Change this from normal nginx setup. Do not add /usr/lib or /lib
+    LD_LIBRARY_PATH=/usr/local/lib:/usr/local/openresty \
+    NGINX_ENVSUBST_OUTPUT_DIR=/usr/local/openresty/nginx/conf \
+    # overridden variables
+    USER=root \
+    # CRS specific variables
+    PARANOIA=1 \
+    ANOMALY_INBOUND=5 \
+    ANOMALY_OUTBOUND=4 \
+    BLOCKING_PARANOIA=1
+
+COPY --from=build /usr/local/modsecurity/lib/libmodsecurity.so.${MODSEC_VERSION} /usr/local/modsecurity/lib/
+COPY --from=build /usr/local/openresty/nginx/modules/ngx_http_modsecurity_module.so /usr/local/openresty/nginx/modules/ngx_http_modsecurity_module.so
+COPY --from=build /usr/share/TLS/server.key /usr/local/openresty/nginx/conf/server.key
+COPY --from=build /usr/share/TLS/server.crt /usr/local/openresty/nginx/conf/server.crt
+COPY --from=build /usr/share/TLS/dhparam-* /etc/ssl/certs/
+COPY --from=build /etc/modsecurity.d/unicode.mapping /etc/modsecurity.d/unicode.mapping
+COPY --from=build /etc/modsecurity.d/modsecurity.conf /etc/modsecurity.d/modsecurity.conf
+COPY --from=crs_release /opt/owasp-crs /opt/owasp-crs
+# We use the templating mechanism from the nginx image here,
+# as set up by owasp/modsecurity-docker
+# Everything from templates except the nginx confiugration is the same for openresty.
+COPY nginx/templates/conf.d /usr/local/openresty/nginx/templates/conf.d
+COPY nginx/templates/includes /usr/local/openresty/nginx/templates/includes
+COPY nginx/templates/modsecurity.d /usr/local/openresty/nginx/templates/modsecurity.d
+# Copy the nginx configuration separately
+COPY openresty/templates/nginx.conf.template /usr/local/openresty/nginx/templates
+# 
+COPY src/etc/modsecurity.d/modsecurity-override.conf /usr/local/openresty/nginx/templates/modsecurity.d/modsecurity-override.conf.template
+COPY src/etc/modsecurity.d/setup.conf /usr/local/openresty/nginx/templates/modsecurity.d/setup.conf.template
+COPY src/bin/healthcheck /usr/local/bin/healthcheck
+COPY openresty/docker-entrypoint.d/*.sh /docker-entrypoint.d/
+COPY src/opt/modsecurity/activate-plugins.sh /docker-entrypoint.d/94-activate-plugins.sh
+COPY src/opt/modsecurity/activate-rules.sh /docker-entrypoint.d/95-activate-rules.sh
+COPY openresty/docker-entrypoint.sh /
```

Comment:
Please move all the Openresty stuff to a separate block after the other `COPY` statements. That will make it easier to maintain and merge changes.